docs/src/manifolds/shapespace.md

```diff
@@ -21,12 +21,6 @@ q = [
 # let's plot them as triples of points on a plane
 fig = scatter(p[1,:], p[2,:], label="p", aspect_ratio=:equal)
 scatter!(fig, q[1,:], q[2,:], label="q")
-
-# aligning q to p
-A = get_orbit_action(M)
-a = optimal_alignment(A, p, q)
-rot_q = apply(A, a, q)
-scatter!(fig, rot_q[1,:], rot_q[2,:], label="q aligned to p")
```

Comment:
This is a relatively important part of shape spaces, so maybe it would be better to move the entire tutorial to LieGroups.jl instead of keeping an incomplete one here.

Reply:
Ah yes, sorry, I did not have that in mind, that it might make the example too minimalistic.

I think it would be great example for ManifoldExamples – maybe even defining the optimal alignment just in the example might work for now.

IN general I think, having more like “how to do this” tutorials here and nice examples over on the examples packages might be a good idea to follow.

Reply:
I understand and sure then lets move it over there.

Reply:
Ok, so then let's just remove code of the example here.